src/assets/simp/build/simp.spec

```diff
@@ -221,7 +222,8 @@ fi
 # Post uninstall stuff
 
 %changelog
-* Tue Jul 17 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - 6.2.0-0
+* Mon Sep 17 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - 6.2.0-0
```

Comment:
Needs to note the following:
* Removal of pupmod-puppetlabs-java_ks dependency from simp package
* Addition of the following dependencies to simp package
  - pupmod-simp-systemd dependency 
  - pupmod-puppetlabs-mount_providers
* Removal of the following dependencies from simp-extras package
  -  pupmod-simp-jenkins 
  -  pupmod-simp-mcafee
* Addition of the following dependencies to simp-extras package
  - pupmod-puppetlabs-docker
  - pupmod-puppetlabs-translate
  - pupmod-simp-dconf
  - pupmod-simp-dirtycow
  - pupmod-simp-simp_docker
  - pupmod-simp-simp_ipa
* Move of the following dependencies from simp to simp-extras
  - pupmod-simp-freeradius
  - pupmod-simp-krb5
* Updates to most dependency versions
~                                                                                          
~